chat-components/src/common/Constants.ts

```diff
@@ -51,4 +51,152 @@ export enum ElementType {
+    public static readonly CloseButtonId = "oc-lcw-header-close-button";
+    public static readonly HeaderIconId = "oc-lcw-header-icon";
+    public static readonly HeaderTitleId = "oc-lcw-header-title";
+    public static readonly HeaderLeftGroupId = "headerLeftGroup";
+    public static readonly HeaderMiddleGroupId = "headerMiddleGroup";
+    public static readonly HeaderRightGroupId = "headerRightGroup";
+    public static readonly DefaultInputValidationPaneId = "oclcw-emailTranscriptDialogContainer";
+    public static readonly DefaultInputValidationPaneInputId = "oclcw-emailTranscriptDialogTextField";
+    public static readonly DefaultLoadingPaneId = "oc-lcw-loadingpane";
+    public static readonly DefaultOOOHPaneId = "oc-lcw-outofofficehours-pane";
+    public static readonly DefaultPostChatSurveyPaneId = "oc-lcw-postchatsurvey-pane";
+    public static readonly DefaultPreChatSurveyPaneId = "oc-lcw-prechatsurveypane-default";
+    public static readonly DefaultProactiveChatPaneId = "oc-lcw-proactivechat";
+    public static readonly DefaultReconnectChatPaneId = "oc-lcw-reconnectchat-pane";
+};
+
+export const ButtonTypes = class {
+    public static readonly Icon = "icon";
+    public static readonly Text = "text";
+};
+
+export const IconNames = class {
+    public static readonly DeclineCall = "DeclineCall";
+    public static readonly Microphone = "Microphone";
+    public static readonly MicOff2 = "MicOff2";
+    public static readonly Video = "Video";
+    public static readonly VideoOff = "VideoOff";
+    public static readonly IncomingCall = "IncomingCall";
+    public static readonly ChromeClose = "ChromeClose";
+    public static readonly Download = "Download";
+    public static readonly Mail = "Mail";
+    public static readonly Volume3 = "Volume3";
+    public static readonly Volume0 = "Volume0";
+    public static readonly ChromeMinimize = "ChromeMinimize";
+};
+
+export const AriaLabels = class {
+    public static readonly EndCall = "End Call";
+    public static readonly MicMute = "Mute";
+    public static readonly MicUnmute = "Unmute";
+    public static readonly VideoTurnCameraOn = "Turn camera on";
+    public static readonly VideoTurnCameraOff = "Turn camera off";
+    public static readonly IncomingCallArea = "Incoming call area";
+    public static readonly RejectCall = "Reject call";
+    public static readonly AcceptVoiceCall = "Accept voice call";
+    public static readonly AcceptVideoCall = "Accept video Call";
+    public static readonly DeclineCall = "Decline Call";
+    public static readonly AudioCall = "Audio Call";
+    public static readonly VideoCall = "Video Call";
+    public static readonly LetsChatWeAreOnline = "Let's chat we are online";
+    public static readonly UnreadMessageString = "you have new messages";
+    public static readonly Close = "Close";
+    public static readonly CloseChat = "Close Chat";
+    public static readonly CancelReturnToChat = "Cancel. Return to Chat";
```

Comment:
Rename to "ConfirmationPaneCancel" and "ConfirmationPaneConfirm"

Reply:
Fixed